bleak/backends/bluezdbus/client.py

```diff
@@ -819,7 +819,7 @@ async def write_gatt_char(
         self,
         characteristic: BleakGATTCharacteristic,
         data: Buffer,
-        response: bool,
+        response: Optional[bool],
```

Comment:
`response` is always a bool when `await self._backend.write_gatt_char(characteristic, data, response)` is called, so I don't think we should be chaning the type hints on any of the backend classes

Reply:
Sorry, you are right that was overshooting by me.